apps/site/src/app/(main)/portal/@applicant/VerticalTimeline.tsx

```diff
@@ -111,6 +112,17 @@ function VerticalTimeline({ status }: VerticalTimelineProps) {
 				/>
 				Attendance confirmed
 			</li>
+		) : status === PortalStatus.void ? (
+			<li className="flex flex-row items-center border-t">
+				<Image
+					src={XCircle}
+					alt="checked-circle"
```

Comment:
Issue: Could you update the `alt` value so that it reflects the `XCircle` instead the `CheckCircle`?

Reply:
Changed `alt` value to `"x-circle"`.